app/src/main/java/org/mozilla/focus/input/InputToolbarIntegration.kt

```diff
@@ -105,9 +123,60 @@ class InputToolbarIntegration(
     override fun start() {
         useCustomDomainProvider = settings.shouldAutocompleteFromCustomDomainList()
         useShippedDomainProvider = settings.shouldAutocompleteFromShippedDomainList()
+        observeStartBrowserCfrVisibility()
```

Comment:
nit: Since the new CFR is to be shown just once we could avoid always observing the store with another `if (showStartBrowsingCfr)` check here.

Reply:
Done